Source/Widgets/CesiumWidget/CesiumWidget.js

```diff
@@ -246,7 +243,7 @@ define([
         this._canRender = false;
         this._renderLoopRunning = false;
         this._showRenderLoopErrors = showRenderLoopErrors;
-        this._resolutionScale = 1.0;
+        this._resolutionScale = window.devicePixelRatio;
```

Comment:
Based on @emackey's [comment](https://github.com/AnalyticalGraphicsInc/cesium/pull/8057#discussion_r314753463) I think this should stay `1.0` and the code above should be

```js
var resolutionScale = widget._resolutionScale;
resolutionScale *= defaultValue(window.devicePixelRatio, 1.0);
```

Given that `resolutionScale` is user controlled I think it's confusing to set any default other than `1.0`.

And if someone wants to achieve the previous behavior they would set `resolutionScale` to `1.0 / window.devicePixelRatio`.

Reply:
I think this is confusing too many concepts. ResolutionScale should always be the resolutions scale. Multiplying it opposed to being a default value insinuates that resolutionScale is some factor of devicePixelRatio.

I am not 100% which solution is right here.